src/lib/json-rpc.ts

```diff
@@ -7,6 +9,10 @@ export interface JsonRpcRequest {
   id?: JsonRpcId;
   method: string;
   params?: JsonRpcParams;
+  /** JSON RPC Subscribe Extension Parameters */
+  subscribe?: { 
```

Comment:
Can we rename this to `subscription` (to a noun).

Reply:
Done